src/Meilisearch/Extensions/ObjectExtensions.cs

```diff
@@ -38,5 +38,31 @@ internal static string ToQueryString(this object source, BindingFlags bindingAtt
             var queryString = string.Join("&", values);
             return queryString;
         }
+
+        /// <summary>
+        /// Transforms a Meilisearch object containing Lists into an URL encoded query string.
+        /// </summary>
+        /// <param name="source">Object to transform.</param>
+        /// <param name="bindingAttr">Binding flags.</param>
+        /// <returns>Returns an url encoded query string.</returns>
+        internal static string ToQueryStringWithList(this object source, BindingFlags bindingAttr = BindingFlags.DeclaredOnly | BindingFlags.Public | BindingFlags.Instance)
```

Comment:
> Instead of creating a List<string>() we can define a Dictionary + System.Net.Http.FormUrlEncodedContent to generate the query string.

I'm not sure to understand what you mean?

>I think we should also prevent this extension from being used incorrectly by defining a more specific type instead of using object, like: internal static string ToQueryString(this IQuery source. And of course we have to create this interface 😬
If you do that, you'll have to change the tests because now all the results will be ascii encoded :)

I think that this one could be dealt with later and for the moment creating an issue will end up not?